src/rocksdb.rs

```diff
@@ -1386,6 +1428,12 @@ impl DB {
         }
     }
 
+    pub fn get_iostall_info(&self, cf: &CFHandle, info: &mut IOStallInfo) -> bool {
```

Comment:
why not return `Option<IOStallInfo>`, I guess it is better for outside use.

Reply:
Good idea. I'll do it.